### DISCUSSION TITLE
Why does V4 schema use a combination of camel case and underscore-case for account/user column names?

### DISCUSSION THREAD

**Discussion by yuklai** [nextauthjs/next-auth#4345] [Help]
https://next-auth.js.org/getting-started/upgrade-v4#schema-changes

kind of bugs me. 
Anyway to pick one or let the app specify it?

**Comment by ardinusawan** [comment]
Bugs me as well, we can use `@map` https://authjs.dev/reference/adapter/prisma#naming-conventions
